backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/v1/controller/UsersController.java

```diff
@@ -50,4 +54,27 @@ public UserModel getUserById(
 		return userModelAssembler.toModel(user);
 	}
 
+
+    @PostMapping({ "/{userId}/email-validations"})
+    @Operation(summary = "Verify the status of a confirmation code.", operationId="verify-confirmation-code")
```

Comment:
The API operation description doesn't really match what the endpoint is doing. I would change this to something like "validate a user's email via a confirmation code" or something like that.

Reply:
Sounds good, implemented in commit 2cebd2d6f0685660fa512c66c4a5da16869070e1.